scopes/ui-foundation/ui/webpack/webpack.base.config.ts

```diff
@@ -348,8 +348,8 @@ export default function createWebpackConfig(
           const entrypointFiles = entrypoints.main.filter((fileName) => !fileName.endsWith('.map'));
 
           return {
-            files: manifestFiles,
-            entrypoints: entrypointFiles,
+            files: JSON.stringify(manifestFiles),
+            entrypoints: JSON.stringify(entrypointFiles),
```

Comment:
why do we need this json stringify here?

Reply:
`WebpackManifestPlugin` now expects `Record<string, string>`. It might have dropped internal `JSON.stringify()`